src/reference/constellation/events.json

```diff
@@ -304,8 +304,11 @@
         "description": "Sent when a user uses direct purchase to buy a title on a Streamer's Channel.",
         "payload": {
             "type":"Object",
-            "description":"An empty JSON Object for now"
-        }
+            "description":"A JSON object containing the title of the game sold."
```

Comment:
Please could you change the `payload` to include `gameTitle` as a property instead? See this for an example: https://github.com/mixer/developers/blob/df0cf202240bd4064463cae2e7c9c351e06b4a48/src/reference/constellation/events.json#L609

Reply:
Sure. Updated like your example.